src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -168,6 +172,19 @@ public override bool Execute()
                 // Summary
                 RestoreSummary.Log(log, restoreSummaries);
 
+                var projectsWithImportsChanged = new StringBuilder();
+                foreach (var summary in restoreSummaries)
+                {
+                    //  Success does not matter here, as long as the files are not changed
+                    if(summary.BuildFilesChanged)
+                    {
+                        projectsWithImportsChanged.Append(summary.InputPath).Append(';');
```

Comment:
would it be easier to just do:
`string.Join(";", restoreSummaries.Where(e => e.BuildFilesChanged).Select(e => e.InputPath))`

instead of the string builder and loops?

Reply:
It is....didn't think too much about it, my instinct was to not add the extra linq overhead lol :)
This is called only once though :D 
So I can change to make it more readable

Reply:
I get avoiding Linq, it's always a trade off it seems. I'm fine with either one.